### DISCUSSION TITLE
How fast api works with pdfplumber?

### DISCUSSION THREAD

**Discussion by Fianax** [jsvine/pdfplumber#1197] [Q&A]
I have a fast api in python to manage the use of pdfplumber through endpoints.

And I would like to know how well pdfplumber works with threads, multithreading, etc.

**Comment by jsvine** [comment]
I'm not aware of any explicit discussions re. FastAPI and `pdfplumber`, but you may find some of these discussions helpful re. threads and multithreading:

- https://github.com/jsvine/pdfplumber/discussions/814
- https://github.com/jsvine/pdfplumber/issues/1059
- https://github.com/jsvine/pdfplumber/issues/995
- https://github.com/jsvine/pdfplumber/issues/91
